public/html/action-popup.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
```

Comment:
Rather than revert `popup-center` from [this change](https://github.com/leather-io/extension/pull/4655/files#diff-d1ec653779941b71087a305b0b7906a6a59a662c2ab702e32fbe526a4250e2fb) 

I instead added this explicitly named  `action-popup` to make it clearer of the usage. 

The file triggered by approvals is now `popup.html` rather than the original `popup-center`. I could be good to name that even more explicitly but I think this could be enough to differentiate more clearly